spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
+require 'rspec/support/object_inspector'
```

Comment:
This require makes `RSpec::Support::ObjectInspector` available in our spec suite, but does not make it available for users of rspec-expectations.  The failing cucumber features demonstrate that; they are failing because it hasn't been loaded in the environment.  It's not the responsibility of each test environment to load this file. Instead, it should be moved into `lib` so that it's always available to anyone who loads rspec-expectations.  I think it should go in `lib/rspec/matchers.rb`.

Also, I'd prefer you use `RSpec::Support.require_rspec_support "object_inspector"`.  That uses an optimized form of require that is more performant on rubies that support `require_relative`.